tests/init.sh

```diff
@@ -314,8 +327,16 @@ run_strace_match_diff()
 
 	run_prog > /dev/null
 	args="$prog_args"
-	run_strace "$@" $args > "$EXP"
-	sed -n "$sed_cmd" < "$LOG" > "$OUT"
+
+	case $ME_ in
+		*@json*)
+			run_strace "$@" $args | python -m json.tool > "$EXP";
+			sed -n "$sed_cmd" < "$LOG" | python -m json.tool > "$OUT" ;;
```

Comment:
I'd recommend saving the output into a file before passing it on to python, this would allow to
1. honor the exit status, and
2. print the output when python fails to process it.

Reply:
Will do, thank you for the advice ! How do you feel about this execution ? I tried to be as unintrusive as possible and the structure tests will trigger only if a `@structure` suffix is appended to the name.

Reply:
The approach to branch on suffixes looks fine. We haven't used `@` symbol in file names before, but I'm not aware of any potential issues with that yet.